src/demo/App.js

```diff
@@ -17,6 +17,19 @@ const stripLocale = (pathname, locale) => {
   return pathname.replace(`/${locale}`, '');
 };
 
+class Search extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
```

Comment:
You don't need the `constructor` function in this case.

When there is no constructor function, it will run `super` automatically. The parent class's `constructor` method (called by `super`) already assigned `this.props`, so there's no need to do it manually.